### PR TITLE
feat: add ru docs for theme

### PR DESCRIPTION
**Pull request by Mister-Hope**

### CODE REVIEW

File: docs/theme/src/.vuepress/navbar/ru.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  {
+    text: "Project",
+    icon: "info",
+    prefix: "/ru/",
+    children: [
+      "/changelog",
+      "/demo",
+      "/contribution",
+      {
+        text: "Plugins",
```
Comment (Mister-Hope) [inline docs/theme/src/.vuepress/navbar/ru.ts:23]: missing translation here

File: docs/theme/src/ru/README.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+@tab TS
+
+```ts
+// .vuepress/config.ts
+import { defineUserConfig } from "vuepress";
+import { hopeTheme } from "vuepress-theme-hope";
+
+export default defineUserConfig({
+  theme: hopeTheme({
+    // your theme config here
```
Comment (Mister-Hope) [inline docs/theme/src/ru/README.md:142]: I think comment like this can be translated

File: docs/theme/src/ru/config/frontmatter/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Frontmatter Config
+icon: config
+index: false
+dir:
+  order: 5
+category:
+  - Config
```
Comment (Mister-Hope) [inline docs/theme/src/ru/config/frontmatter/README.md:8]: I think you can translate tags and categories into russian

File: docs/theme/src/ru/config/frontmatter/info.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
+---
```
Comment (Mister-Hope) [inline docs/theme/src/ru/config/frontmatter/info.md:1]: Missing translation

File: docs/theme/src/ru/config/frontmatter/layout.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
+---
```
Comment (Mister-Hope) [inline docs/theme/src/ru/config/frontmatter/layout.md:1]: Missing translation

File: docs/theme/src/ru/config/frontmatter/plugins.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
+---
```
Comment (Mister-Hope) [inline docs/theme/src/ru/config/frontmatter/plugins.md:1]: Missing translation

File: docs/theme/src/ru/config/i18n.md [hunk 7]
```diff
@@ -0,0 +1,189 @@
+---
```
Comment (Mister-Hope) [inline docs/theme/src/ru/config/i18n.md:1]: Missing translation

**Comment by Mister-Hope** [comment]
Just have a quick review, a lot of files are still missing it's translation.

So you just complete `guide` right?

**Review by Mister-Hope — COMMENTED** [review]
@Hkr32

**Comment by Hkr32** [comment]
Hi. I only translated the guide) There is still a lot to translate... There is not so much free time. Schedule as you like, I will translate according to my strength)

File: README.md [hunk 8]
```diff
@@ -3,9 +3,15 @@
   <img width="240" src="https://vuepress-theme-hope.github.io/v2/logo.svg" style="text-align: center;">
 </p>
 <h1 align="center">vuepress-theme-hope</h1>
-<h4 align="center">A vuepress theme with tons of features✨ / 一个具有强大功能的 vuepress 主题✨</h4>
+<h4 align="center">A vuepress theme with tons of features✨</h4>
 
-[![Author: Mr.Hope](https://img.shields.io/badge/作者-Mr.Hope-blue.svg?style=for-the-badge)](https://mrhope.site)
+<h4 align="center">
+
+[中文简介](README-zh.md) | [Russian README](README-ru.md)
```
[the remark this answers is not in the document]
Comment (Hkr32) [reply README.md:10]: Everything is translated there. Did I not notice something?

**Comment by Mister-Hope** [comment]
Sorry for the delay, really busy in the past few days, I updated your permission, and you should be able to have "Write" access to all repos in this organization.

**Comment by Hkr32** [comment]
I finished. If you have questions, write.

**Comment by Mister-Hope** [comment]
Maybe try to fix the linter?

File: docs/theme/src/ru/config/plugins/pwa.md [hunk 9]
```diff
@@ -0,0 +1,306 @@
...
+
+Мы рекомендуем вам установить favicon для вашего сайта
+
+:::
+
+### themeColor
+
+- 类型: `string`
+- 默认值: `"#46bd87"`
+
```
Comment (Mister-Hope) [inline docs/theme/src/ru/config/plugins/pwa.md:89]: Here?

**Comment by Hkr32** [comment]
I fixed it. What else?

**Comment by Mister-Hope** [comment]
Thanks for the great work

**Pull request merged**